meson.build

```diff
@@ -166,31 +166,20 @@ systemd_system_unit_dir = dependency('systemd').get_variable(
 udev_dir = dependency('udev').get_variable(
         'udev_dir')
 
-install_data(
-    'ibm_vpd/system-vpd.service',
-    install_dir: systemd_system_unit_dir)
+rules = ['service_files/70-ibm-vpd-parser.rules']
 
-install_data(
-    'ibm_vpd/ibm-vpd-parser@.service',
-    install_dir: systemd_system_unit_dir)
+services = ['service_files/system-vpd.service', 'service_files/ibm-vpd-parser@.service', 'service_files/com.ibm.VPD.Manager.service', 'service_files/wait-vpd-parsers.service', 'service_files/ibm-isdimm-vpd-parser@.service', 'service_files/ibm-spi-vpd-parser@.service']
 
-install_data(
-    'ibm_vpd/com.ibm.VPD.Manager.service',
-    install_dir: systemd_system_unit_dir)
+scripts = ['scripts/wait-vpd-parsers.sh']
 
-install_data(
-    'ibm_vpd/wait-vpd-parsers.service',
-    install_dir: systemd_system_unit_dir)
+install_data(rules, install_dir: udev_dir / 'rules.d')
```

Comment:
move these install files just after the place where the files are defined

Reply:
done